main.go

```diff
@@ -122,15 +123,15 @@ func main() {
 		}
 		log.SetLevel(level)
 
-		switch c.String("log-format") {
+		switch strings.ToLower(c.String("log-format")) {
 		case "text":
 			log.SetFormatter(&log.TextFormatter{FullTimestamp: true})
 		case "json":
 			log.SetFormatter(&log.JSONFormatter{})
```

Comment:
we should use the following formatter to make sure its compatible internally

```
&log.JSONFormatter{
		FieldMap: log.FieldMap{
			log.FieldKeyTime:  "@timestamp",
			log.FieldKeyLevel: "@level",
			log.FieldKeyMsg:   "@message",
		},
	}
```

Reply:
added